tests/suites/static_analysis/lint_go.sh

```diff
@@ -108,14 +131,19 @@ test_static_analysis_go() {
 
     ## go vet, if it exists
     if go help vet >/dev/null 2>&1; then
-      run "run_go_vet"
+      run "run_go_vet" "${FOLDERS}"
     fi
 
     ## golint
     if which golint >/dev/null 2>&1; then
       run "run_go_lint"
     fi
 
+    ## goimports
+    if which goimports >/dev/null 2>&1; then
+      run "run_go_imports" "${FOLDERS}"
+    fi
```

Comment:
It would be nice if these had output to mention they are being skipped.

Reply:
They do, the run command outputs if they're skipped 😉

Reply:
Yeah, sorry, I get what you mean - yes, I'll change this in a future PR. I did it so it was optional at first, but we've gone all in, so we should just be explicit.